project/Dependencies.scala

```diff
@@ -268,4 +268,14 @@ object Dependencies {
     sam,
     leonardo
   )
+
+  val pact4sV = "0.7.0"
+  val pact4sScalaTest = "io.github.jbwheatley" %% "pact4s-scalatest" % pact4sV % Test
+  val pact4sCirce = "io.github.jbwheatley" %% "pact4s-circe" % pact4sV
+
+  val pact4sDependencies = Seq(
+    pact4sScalaTest,
+    pact4sCirce,
+    logbackClassic
```

Comment:
No need to add `logbackClassic` here. Replacing `logback.xml` by `logback-test.xml` should work.

Reply:
Thanks, that worked!